stock_unfuck/models/res_config.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+
+
+class StockConfiguration(models.TransientModel):
+
+    _inherit = 'stock.config.settings'
+
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True,
+        default=lambda self: self.env.user.company_id)
+
+    make_journal_entry = fields.Boolean(
```

Comment:
@hbto 

Are you agree in add this change in this module?

I thought that this configuration it would be into product_extended_variants module